### ISSUE TITLE
browser.pages() doesn't return pages that are opened manually (via browser UI)

### ISSUE THREAD

**Issue by bobajeff** [mattsse/chromiumoxide#24]
Running the following `main.rs` file will only print `1` *(the one opened via `browser.new_page`)* even if other pages are already open or new ones get opened manually through the browser UI:
```rust
use futures::StreamExt;
use chromiumoxide::error::Result;
use chromiumoxide::browser::Browser;

mod get_debug_ws_url;

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let debug_ws_url = get_debug_ws_url::get_ws_url().await?;
    let (browser, mut handler) = 
    Browser::connect(debug_ws_url).await?;

    let handle = async_std::task::spawn(async move {
        loop {
            let _ = handler.next().await.unwrap();
        }
    });

    let my_handle = async_std::task::spawn(async move {
        let page = browser.new_page("https://en.wikipedia.org").await;
        loop {
            let pages = browser.pages().await;
            let pages = pages.unwrap_or_default();
            println!("{:?}", pages.len());
            async_std::task::sleep(std::time::Duration::from_secs(5)).await;
        }
    });
        handle.await;
    Ok(())
}
```
Edit: Also, this is the `get_debug_ws_url.rs`:
```rust
use std::collections::HashMap;

pub async fn get_ws_url() -> Result<std::string::String, Box<dyn std::error::Error>> {
    let resp = reqwest::get("http://127.0.0.1:9222/json/version")
        .await?
        .json::<HashMap<String, String>>()
        .await?;
        let web_socket_debugger_url = resp["webSocketDebuggerUrl"].clone();
    Ok(web_socket_debugger_url)
}
```

**Comment by mattsse** [comment]
I think the reason for this is, that currently only user initiated pages are tracked in `Handler`. 
I remember there was an issue when initializing all targets during start up did not work so I restricted it to only those targets initiated by a `browser::new_page` command. 
This shouldn't be to hard to fix, since the handler already knows about all the existing targets, they are just not properly initialized and won't be returned via a `get_pages` request 
I'll have a look.

**Comment by mattsse** [comment]
I believe I fixed this with #26 
Can you confirm or still the wrong amount of pages?

**Comment by bobajeff** [comment]
> I believe I fixed this with #26
> Can you confirm or still the wrong amount of pages?

It's fixed. I've checked it's reports the correct about of tabs. Thanks.
